src/subdomains/generic/user/models/user/user.entity.ts

```diff
@@ -20,11 +20,24 @@ export enum UserStatus {
   DELETED = 'Deleted',
 }
 
+export enum UserAddressType {
+  BITCOIN_LEGACY = 'BitcoinLegacy',
+  BITCOIN_BECH32 = 'BitcoinBech32',
+  EVM = 'EVM',
+  LNNID = 'LNNID',
+  FIAT = 'Fiat',
+  LIGHTNING = 'Lightning',
+  UMA = 'UMA',
+}
```

Comment:
- Remove `FIAT`
- Use values from `LightningAddressType` (`src/integration/lightning/lightning-helper.ts`) instead of `LNNID` and `LIGHTNING`
- Add Monero, Liquid, Arweave and Cardano